bentoml/yatai/client/bento_repository_api.py

```diff
@@ -226,14 +310,12 @@ def upload_from_dir(self, saved_bento_path, labels=None):
                 tar.add(saved_bento_path, arcname=bento_service_metadata.name)
             fileobj.seek(0, 0)
 
-            if response.uri.type == BentoUri.S3:
-                http_response = requests.put(
-                    response.uri.s3_presigned_url, data=fileobj
-                )
-            elif response.uri.type == BentoUri.GCS:
-                http_response = requests.put(
-                    response.uri.gcs_presigned_url, data=fileobj
-                )
+            upload_path = (
```

Comment:
why this change?  it seems better previously - except that it should also handle the "else" case and report error

Reply:
It feels redundant.  I can revert the change